invoices/invoiceregistry.go

```diff
@@ -676,13 +676,23 @@ func (i *InvoiceRegistry) processKeySend(ctx invoiceUpdateCtx) error {
 		return errors.New("final expiry too soon")
 	}
 
+	// The invoice database indexes all invoices by payment address, however
+	// legacy keysend payment do not have one. In order to avoid a new
+	// payment type on-disk wrt. to indexing, we'll continue to insert a
+	// blank payment address which is special cased in the insertion logic
+	// to not be indexed. In the future, once AMP is merged, this should be
+	// replaced by generating a random payment address on the behalf of the
+	// sender.
+	payAddr := channeldb.LegacyKeysendPayAddr
```

Comment:
This addition is not functional and just serves as clarification right? I am ok with that.

Reply:
yes, sort of a place holder until the final random pay addr is generated 👍